marlowe-dashboard-client/src/Component/Contract/View.purs

```diff
@@ -8,26 +8,33 @@ module Component.Contract.View
 
```

Comment:
Main changes in this file:

1. Passing `RoleTokenStore` instead of `ContractUserParties`
2. Rendering the addresses and adding click-to-copy